webview-ui/src/Periscope/ErrorView.tsx

```diff
@@ -0,0 +1,39 @@
+import { VSCodeDivider } from "@vscode/webview-ui-toolkit/react";
+import { PeriscopeTypes } from "../../../src/webview-contract/webviewTypes";
+import styles from "./Periscope.module.css";
+
+export interface ErrorViewProps {
+    clusterName: string
+    message: string
+    config: PeriscopeTypes.KustomizeConfig
+}
+
+export function ErrorView(props: ErrorViewProps) {
+    return (
+        <>
+            <i className={[styles.errorIndicator, "fa", "fa-times-circle"].join(" ")}></i>
```

Comment:
I don't love this syntax, and I've ended up using it a few times in this PR. It might be worth using the [`classnames` package](https://github.com/JedWatson/classnames) to clean this up.

Reply:
I'll submit a separate PR for this